mdoc-sbt/src/main/scala/mdoc/MdocPlugin.scala

```diff
@@ -186,7 +186,7 @@ object MdocPlugin extends AutoPlugin {
         fullClasspath.in(ref, Compile).value.map(_.data),
         classloadedSetting(
           ref,
-          "org.scalajs.core.tools.linker.backend.ModuleKind",
+          "org.scalajs.linker.interface.ModuleKind",
           "scalaJSModuleKind"
```

Comment:
This code is trying to read the value of the `scalaJSModuleKind` setting via reflection so that we can avoid a dependency on the sbt-scalajs plugin from sbt-mdoc. We may be able to achieve the same thing like this
```
        classloadedSetting(
          ref,
          "org.scalajs.linker.interface.StandardConfig",
          "scalaJSLinkerConfig"
        ).value.map { config =>
          config.getClass.getMethod("moduleKind").invoke(config).toString
        }
```